### PR TITLE
use moment for timestamps to use correct time zone

### PR DESCRIPTION
**Pull request by ethatcher83**
use moment for timestamps to use correct time zone

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "moment": "^2.26.0",
```
Comment (ghost) [inline package.json]: change to ~2.26.0 to protect from breaking changes

File: src/MiddlewareFactory.ts [hunk 2]
```diff
@@ -3,22 +3,23 @@ import { RequestHandler } from "express";
 import StructuredLogger from "./StructuredLogger";
 import getRequestMetaData from "./getRequestMetaData";
 import onFinished from "on-finished";
+import moment from 'moment';
```
Comment (ghost) [inline src/MiddlewareFactory.ts]: use double quotes

File: src/formatting/HumanFormatFactory.ts [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import LogMetaData from "../LogMetaData";
 import winston from "winston";
+import moment from 'moment';
```
Comment (ghost) [inline src/formatting/HumanFormatFactory.ts]: Use double quotes

File: src/MiddlewareFactory.ts [hunk 4]
```diff
@@ -3,22 +3,23 @@ import { RequestHandler } from "express";
...
+            const startTime = moment().format();
 
             const metaData = getRequestMetaData(request);
 
             logger.request("Start requesting", metaData);
 
             onFinished(response, function () {
 
-                const duration = Date.now() - startTime;
+                const duration = moment().format(); - startTime;
```
Comment (ghost) [inline src/MiddlewareFactory.ts]: This is not going to work

File: src/MiddlewareFactory.ts [hunk 5]
```diff
@@ -3,22 +3,23 @@ import { RequestHandler } from "express";
...
+import moment from 'moment';
 
 class MiddlewareFactory {
 
     public static create(logger: StructuredLogger) {
 
         const middleware: RequestHandler = function (request, response, next) {
 
-            const startTime = Date.now();
+            const startTime = moment().format();
```
Comment (ghost) [inline src/MiddlewareFactory.ts]: this needs to just be moment()

**Review by nabeelamirch — APPROVED** [review]

**Pull request merged**